format

```diff
@@ -0,0 +1,2 @@
+refmt --in-place --print-width 80 RNTester/src/**/*.re
+refmt --in-place --print-width 80 src/**/*.re
```

Comment:
Will this cover `.rei` as well? 

BTW, there's a sense to add contributing guidelines with `precommit` hook instructions.